snapcraft/internal/repo/_platform.py

```diff
@@ -26,6 +26,7 @@
     '"elementary"',
     'debian',
     'neon',
+    'LinuxMint',
```

Comment:
It's not hard to write a replacement function to just process the `os-release(5)` file. I did it some time ago. I can submit a PR to replace the function call with direct parsing of the file.